.github/ISSUE_TEMPLATE/z_release_major.md

```diff
@@ -18,35 +18,35 @@ assignees: ''
 
 When development is ready to begin one of the engineers should be nominated as a Release Engineer. They will be responsible for making sure the following tasks are completed though not necessarily completing them.
 
-- [ ] Set the version number in the `package.json` and `package-lock.json`, then submit a PR to `master` branch. The easiest way to do this is to use `npm --no-git-tag-version version <major|minor>`.
-- [ ] Ensure that issues from merged commits are closed and mapped to a milestone.
+- [ ] Checkout to a new `<major>.<minor>.<patch>-release` branch (eg 4.4.x-release) and set the version number in the `package.json` and `package-lock.json`. The easiest way to do this is to use `npm --no-git-tag-version version <major|minor>`. Once the version is updated, submit a PR to `master` branch.
+- [ ] Ensure that issues associated with merged commits to `master` are closed and mapped to a milestone.
 
 # Releasing - Release Engineer
 
-Once all issues have been merged into `master`, and the `master` branch has the new version number, then the release process can start:
+Once the PR has been merged into `master`, and the `master` branch has the new version number, then the release process can start:
```

Comment:
I prefer the old wording. The new wording only talks about the version change PR, whereas the old wording is talking about ALL the issues in the milestone.